testing/object-utils.ts

```diff
@@ -1,9 +0,0 @@
-// Performs a deep copy of the given object. If there are cycles in the object
```

Comment:
I'm not certain we want to move this into `Perseus-core`. It's only used in tests. Also, if we want to do this move, I'd want to see if we can consolidate all the different ways we clone (today, we have this `clone`, then we have [`deepClone()`](https://github.com/Khan/perseus/blob/341d316aa8727ebb9e7fde28fc4e2d8779aa3e82/packages/perseus/src/util.ts#L718), another [`clone()`](https://github.com/Khan/perseus/blob/f5a2cf521291180dbbd448adc97700f7c52c8b50/packages/perseus-linter/src/__tests__/tree-transformer.test.ts#L6), and [`_.clone()`](https://github.com/Khan/perseus/blob/368e222a6577dff38143d1584d6773129e8abbd7/packages/kas/src/nodes.ts#L573): all of which look to do very similar things).

Reply:
This was a total mistake from me being on auto-pilot. Thanks for catching it.